qc-runner/src/dataIntegration.r

```diff
@@ -133,7 +133,11 @@ run_dataIntegration <- function(scdata, config){
           # Should we still continue if data is not integrated? No, right now..
           print("Current number of cells per sample: ")
           print(table(scdata$samples))
-          stop("Error thrown in IntegrateData: Probably one/many of the samples contain to few cells.\nRule of thumb is that this can happen at around < 100 cells.")
+          print("Error thrown in IntegrateData: Probably one/many of the samples contain to few cells.\nRule of thumb is that this can happen at around < 100 cells.")
```

Comment:
```suggestion
          warning("Error thrown in IntegrateData: Probably one/many of the samples contain to few cells.\nRule of thumb is that this can happen at around < 100 cells.")
```